pyproject.toml

```diff
@@ -24,10 +24,12 @@ mypy = "^0.812"
 # Reasons disabled:
 # too-many-* - not enforced for the sake of readability
 # too-few-* - same as too-many-*
+# unexpected-keyword-arg - argues that glocaltokens has no verbose argument
```

Comment:
It's a valid complain. You need to update `glocaltokens` version in poetry as well.

Reply:
yeh, it felt like i m missing something here :D

Reply:
Fixed ✅